academic_helper/logic/schedule_export.py

```diff
@@ -0,0 +1,114 @@
+from academic_helper.models import ClassSchedule
+import arrow
+from ics import Calendar, Event
+
+
+# TODO: needs to be modeled in the DB
+_HUJI_SEMESTER_DATES = {
+    1: {
+        "begin": arrow.get(year=2019, month=10, day=27),
+        "end": arrow.get(year=2020, month=1, day=28),
```

Comment:
Why using arrow? No need for an external library for that, right?

Reply:
Good point - originally I thought it will have more usage than the standard datetime. Generally `arrow` is much more flexiable and feature-rich than `datetime` IMO.

Anyways - the only thing that avoids removing it is the `.shift(weekday=)` functionality - I don't think `datetime`/`timedelta` provides that out-of-the-box, see line 100 in this file.

I can try working around it - let me know what you prfer.

Reply:
Ok. I didn't really dive into the logic. If you think its better this way, leave it like that.